src/ResourceManager/Network/Commands.Network/Cortex/HubVnetConnection/GetAzureRmHubVirtualNetworkConnectionCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using Microsoft.Azure.Commands.Network.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Network.Models;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+
+    [Cmdlet(VerbsCommon.Get,
+        "AzureRmHubVirtualNetworkConnection",
```

Comment:
Needs new prefix naming structure: VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HubVirtualNetworkConnection"

This will make both the Netcore and Desktop build work correctly.  Please do for all cmdlets.

Reply:
Fixed